engine/src/main/java/org/terasology/rendering/world/WorldRendererImpl.java

```diff
@@ -326,7 +335,7 @@ private void initRenderGraph() {
         renderGraph.addNode(initialPostProcessingNode, "initialPostProcessingNode");
 
         aLabel = "downSampling_gBuffer_to_16x16px_forExposure";
-        DownSamplerForExposureNode exposureDownSamplerTo16pixels = new DownSamplerForExposureNode(context, sceneOpaqueFboConfig, displayResolutionDependentFBOs, FBO_16X16_CONFIG, immutableFBOs, aLabel);
+        DownSamplerForExposureNode exposureDownSamplerTo16pixels = new DownSamplerForExposureNode(context, writeOnlyGBufferConfig, displayResolutionDependentFBOs, FBO_16X16_CONFIG, immutableFBOs, aLabel);
```

Comment:
Follow up from previous comment, a few lines above: for example who is to say that in here we don't need the content of the readOnlyGBuffer instead? And in fact in this case theoretically we should because this node only reads from the gBuffer, and it should have the latest content from it.